### PR TITLE
Add raspberrypi-debian base image which is based on Debian armel

### PR DESCRIPTION
**Pull request by nghiant2710**

### CODE REVIEW

**Comment by nghiant2710** [comment]
@petrosagg please review this PR

File: device-base/generate-dockerfile.sh [hunk 1]
```diff
@@ -138,39 +140,37 @@ for device in $devices; do
 
 	# Debian.
 	debian_dockerfilePath="$device/debian"
-	if [ $device != "raspberrypi" ]; then
```
Comment (Page-) [inline device-base/generate-dockerfile.sh]: I love that it lets us get rid of this if :)

**Comment by petrosagg** [comment]
@nghiant2710 can you alias it to raspbian instead? I give up purity

**Comment by nghiant2710** [comment]
Close since this PR will be moved to resin-rpi-raspbian repo

**Pull request closed without merging**